homeassistant/components/assist_satellite/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""Base class for assist satellite entities."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+from .entity import AssistSatelliteEntity
+from .models import AssistSatelliteState
+
+__all__ = [
+    "DOMAIN",
+    "AssistSatelliteState",
+    "AssistSatelliteEntity",
+]
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = cv.PLATFORM_SCHEMA
```

Comment:
we shouldn't support platforms, just config entries, do we still add this ?

Reply:
Removed. hassfest seems fine with it.